examples/python-examples/MeshOffsetWeighted.dox.py

```diff
@@ -0,0 +1,27 @@
+from meshlib import mrmeshpy
+
+# Create some mesh
+mesh = mrmeshpy.makeTorus(primaryRadius=1.0,
+                          secondaryRadius=0.1,
+                          primaryResolution=16,
+                          secondaryResolution=16)
+verts_num = mesh.points.size()
+
+# Create VertScalars obj with weights for every vertice
+scalars = mrmeshpy.VertScalars(verts_num)
+for i in range(verts_num):
+    weight = abs(mesh.points.vec[i].x / 5) + 0.1 # Offset size for point
+    scalars.vec[i] = weight
+
+# ===params
+
+params = mrmeshpy.WeightedPointsShellParameters()
+params.offset = 0.1 # common offset that will be added for every point to weighted
```

Comment:
in comment it is better to say that offset is a threshold on distance not to confuse it with point weights